internal/action/sync_test.go

```diff
@@ -30,6 +30,7 @@ func TestSync(t *testing.T) { //nolint:paralleltest
 	act, err := newMock(ctx, u.StoreDir(""))
 	require.NoError(t, err)
 	require.NotNil(t, act)
+	ctx = act.cfg.WithConfig(ctx)
```

Comment:
I guess this answers my above question, and so no we cannot remove the cfg from the action since we use it to populate the contexts...

Reply:
Ack.